simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/station/component/exception/EnabledVariableAccessor.java

```diff
@@ -17,6 +17,7 @@
 public class EnabledVariableAccessor extends VariableAccessor {
 
     public static final String NAME = "Enabled";
+    public static final String EVSE_ENABLED_STATUS = Boolean.TRUE.toString();
```

Comment:
This is a non java dependent constant right? So it should be a string literal?

Reply:
just helps to avoid any spelling mistakes

Reply:
But the constant you are referencing is independent of the java's boolean toString right? They just happen to be the same. If java change's their "true" boolean to be "True" tomorrow (not likely, but still), your constant will break.